Networking/Networking/Remote/RefundsRemote.swift

```diff
@@ -47,7 +70,7 @@ public final class RefundsRemote: Remote {
                            orderID: Int,
                            refundID: Int,
                            completion: @escaping (Refund?, Error?) -> Void) {
-        let path = "\(Path.orders)/" + String(orderID) + "/" + "\(Path.refunds)"
+        let path = Path.orders + "/" + String(orderID) + "/" + Path.refunds + "/" + String(refundID)
```

Comment:
This still isn't terribly readable. Is there a way to convert the orderID and refundID to String, while maintaining the single line? Or do I pull those out and make them separate variables?